properties-panel-extension/README.md

```diff
@@ -73,60 +67,77 @@ function MagicPropertiesProvider(propertiesPanel, translate) {
 
 ### Define a Group
 
-As part of the properties provider we define the groups for the magic tab, too:
+As part of the properties provider we define the magic group:
 
 ```javascript
 // Require your custom property entries.
 // The entry is a text input field with logic attached to create,
 // update and delete the "spell" property.
-var spellProps = require('./parts/SpellProps');
+import spellProps from './parts/SpellProps';
 
-// Create the custom magic tab
-function createMagicTabGroups(element, translate) {
+// Create the custom magic group
+function createMagicGroup(element, translate) {
 
-  // Create a group called "Black Magic".
-  var blackMagicGroup = {
-    id: 'black-magic',
-    label: 'Black Magic',
-    entries: []
+  // create a group called "Magic properties".
+  var magicGroup = {
```

Comment:
We are using `const` already in various places in this example, hence I propose to also use it here